psamm/commands/gapfill.py

```diff
@@ -99,3 +110,9 @@ def run(self):
                 print('{}\t{}\t{}'.format(rxnid, 'Reverse', rxt))
         else:
             logger.info('No blocked compounds found')
+
+    def _log_epsilon_and_fail(self, epsilon, exc):
+        msg = ('Finding blocked compounds failed with epsilon set to {}. Try'
+               ' lowering the epsilon value to reduce artifical constraints on'
+               ' the model.'.format(epsilon))
+        self.fail(msg, exc)
```

Comment:
Yes, that's exactly my rationale for making this a failure. The user has specified the epsilon value correctly but something internally in the command caused a failure.